Automattic/woocommerce-payments
# Allow to upload files as platform account

Fixes #3730 

#### Changes proposed in this Pull Request

Upload files for branding not as accounts but platform. 

Slack discussion: [p1638277763489000-slack-C02C00HCJ8J
](p1638277763489000-slack-C02C00HCJ8J)

Update the _wcpay_feature_card_readers option to 1.
Update the card_present_eligible to true.
Update isBrandingEnabled feature flag to true in client/card-readers/settings/index.js.
Navigate to Payments -> Card Readers page -> Receipt details.
Scroll down to Branding and upload image file to Logo field.
Save settings and even saved successfully the Logo setting is empty.

#### Testing instructions

First set up the stage:
1. Checkout at 1584-gh-Automattic/woocommerce-payments-server branch
1. Update the _wcpay_feature_card_readers option to 1.
2. Update isBrandingEnabled feature flag to true in client/card-readers/settings/index.js.
3. Update the `card_present_eligible` to `true`. (Response from server)

Test:
1. Navigate to Payments -> Card Readers page -> Receipt details.
2. Scroll down to Branding and upload image file to Logo field.
3. Scroll down to Branding and upload image file to Icon field.
4. Save settings
5. Refresh and observe setting are saved. 

-------------------
